terraform/modules/azure_dns/inputs-required.tf

```diff
@@ -0,0 +1,16 @@
+
+variable "dns_resource_group" {
+  type        = string
+  description = "The name of the resource group that contains the DNS zone where the records will be added."
+  default     = "reformmgmtrg"
+}
+
+variable "zone_name" {
+  type        = string
+  description = "The zone name for the CNAME Record"
+  default     = "platform.hmcts.net"
```

Comment:
🚫 the zone name is not always platform.hmcts.net.

this should likely not have a default and be explicit to prevent mistakes.

also did you check all the prod domains or just put something in? as I saw cmc one was wrong and I'm pretty sure there's other service.gov.uk domains or justice.gov.uk domains in use.

Reply:
Sure - no problem, I can remove the default value but most of the domains did have above as zone so thought its better to have defuault value here and then only send custom domain for the acounts which needs it.
For cmc account, we don't seem to have account setup on production but for non-prod, it was definitely above zone.  I thought I would create prod account as well hence added prod value.    I have checked all the prod and non-prod domains and they all are real ones rather than just a place holders :)

Reply:
![image](https://user-images.githubusercontent.com/21194782/221549662-7bcb50ce-489e-4c20-8e44-f2d1efe9ba1b.png)

https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/8999dec3-0104-4a27-94ee-6588559729d1/resourceGroups/SendGrid-prod/providers/Microsoft.SaaS/resources/cmc-prod/overview

CMC must have their prod one on the old system